### ISSUE TITLE
api_mock_gpu_canvas is broken with AFL

### ISSUE THREAD

**Issue by jonathanmetzman** [google/oss-fuzz#1338]
The [logs](https://pantheon.corp.google.com/storage/browser/skia-logs.clusterfuzz-external.appspot.com/afl_skia_api_mock_gpu_canvas/afl_asan_skia/2018-04-17) show this output:
```
[0;36mafl-fuzz [1;97m2.52b[0m by <lcamtuf@google.com>
[1;92m[+] [0mLooks like we're not running on a tty, so I'll be a bit less verbose.[0m
[1;92m[+] [0mYou have 1 CPU core and 1 runnable tasks (utilization: 100%).[0m
[1;94m[*] [0mChecking core_pattern...[0m
[1;94m[*] [0mSetting up output directories...[0m
[1;92m[+] [0mOutput directory exists but deemed OK to reuse.[0m
[1;94m[*] [0mDeleting old session data...[0m
[1;92m[+] [0mOutput dir cleanup successful.[0m
[1;94m[*] [0mScanning '/afl_input_dir'...[0m
[1;92m[+] [0mNo auto-generated dictionary tokens to reuse.[0m
[1;94m[*] [0mCreating hard links for all input files...[0m
[1;94m[*] [0mValidating target binary...[0m
[1;92m[+] [0m[1;95mPersistent mode binary detected.[0m
[1;92m[+] [0m[1;95mDeferred forkserver binary detected.[0m
[1;94m[*] [0mAttempting dry run with 'id:000000,orig:0000000000000048'...[0m
[1;94m[*] [0mSpinning up the fork server...[0m

[1;91m[-] [0mHmm, looks like the target binary terminated before we could complete a
    handshake with the injected code. Perhaps there is a horrible bug in the
    fuzzer. Poke <lcamtuf@coredump.cx> for troubleshooting tips.
)B[?25h[0m[1;91m
[-] PROGRAM ABORT : [1;97mFork server handshake failed[1;91m
         Location : [0minit_forkserver(), afl-fuzz.c:2253
``` 

I suspect this is related to https://bugs.chromium.org/p/chromium/issues/detail?id=803513 but it is hard to tell.
I have a simple fix up for review that deals with that issue https://reviews.llvm.org/D45744
I'll investigate further if this problem persists after my fix lands.

**Comment by jonathanmetzman** [comment]
cc @kjlubick

**Comment by inferno-chromium** [comment]
Note that https://github.com/google/oss-fuzz/pull/1381 will break skia build today, so lets disable this fuzz target for afl.

**Comment by inferno-chromium** [comment]
Actually ignore last comment, i have another workaround.

**Comment by inferno-chromium** [comment]
Workaround is this - https://github.com/google/oss-fuzz/pull/1381/commits/491776ef50f3aef5262bf6c900ad41ee770f29d1. @jonathanmetzman will revert that later when we have a fix for this one.

**Comment by jonathanmetzman** [comment]
I've disabled this fuzzer in #1383

**Comment by jonathanmetzman** [comment]
I've tried building the fuzzer again after my fix for AFL landed, it still seems broken. I suspect patchelf is causing the breakage but still unsure

**Comment by jonathanmetzman** [comment]
Closed by #1576 
patchelf is no longer used, this is no longer a problem
